docs/sources/reference/components/prometheus.exporter.snmp.md

```diff
@@ -187,13 +198,79 @@ prometheus.remote_write "demo" {
 }
 ```
 
+This example uses the alternative way to pass targets:
```

Comment:
I would be in favor of just updating the existing example[s] since we plan to deprecate the old way. Someone can still find/use them that way but better to not model it for them.

Reply:
I'm afraid to scare users if I advertise it as deprecated now. We just made some big changes with Alloy and that upsets some people. That's why I put it as "an alternative way of doing it" for now and kept the old way with examples so that users are not lost. The next major release is far in the future, maybe we should only start deprecating things once we arrive close to it?